main.js

```diff
@@ -220,18 +224,41 @@ document.querySelector('#hogCardContainer').addEventListener("click", (e)=> {
   const form= document.querySelector('form');
   form.addEventListener('submit',(e) => {
     e.preventDefault();
+    // const changeName = () => {
```

Comment:
To keep our code clean, when you create a PR (mostly when you're working with a team) we want to make sure all commented out code that is not for documentation reasons, has been removed. If you need it as an example, you can always add it to the readme.